tests/sentry/integrations/github/test_integration.py

```diff
@@ -585,16 +586,24 @@ def test_github_prevent_install_until_pending_deletion_is_complete(self):
             organization=self.organization, integration=integration
         ).exists()
 
-    def set_rate_limit(self, remaining, limit=5000):
-        """Helper class to set the rate limit"""
-        responses.add(
-            method=responses.GET,
-            url="https://api.github.com/rate_limit",
-            json={
+    def set_rate_limit(
+        self, remaining=MINIMUM_REQUESTS + 100, limit=5000, json_body=None, status=200
```

Comment:
I'm adding `json_body` and `status` to support the 401 bad request message.

I'm also changing `remaining` to a value that simplifies the code that calls this.